app/controllers/trays_controller.rb

```
@@ -228,4 +192,66 @@ def withdraw
   def missing
     @tray = Tray.find(params[:id])
   end
+
+  def invalid
+    @tray = Tray.find(params[:id])
+    @thickness = params[:thickness]
+    @barcode = params[:barcode]
+    @id = params[:user_id]
+  end
+
+  def create_item(tray=Tray.find(params[:id]), barcode=params[:barcode], id=params[:user_id], thickness=params[:thickness])
+    begin
+      item = GetItemFromBarcode.call(barcode: barcode, user_id: id)
+    rescue StandardError
```

My reservation about capturing the exception this way is that it will capture ALL exceptions, and other valid exceptions may go unnoticed. I'm wondering if we shouldn't make this rescue more specific to the type of exception we really want to trap and let the other bubble up.